src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -1160,6 +1162,39 @@ static inline void verify_args(const ASR::IntrinsicScalarFunction_t& x,
 
 } // namespace BinaryIntrinsicFunction
 
+
+namespace ObjectType {
+
+     static inline void verify_args(const ASR::IntrinsicScalarFunction_t& x, diag::Diagnostics& diagnostics) {
+        ASRUtils::require_impl(x.n_args != 1,
```

Comment:
```suggestion
        ASRUtils::require_impl(x.n_args == 1,
```

Reply:
Hmm.. this check was really confusing. It was a bit difficult to understand what we were checking. Thanks a lot for this! :+1: